src/components/MyKiva/BadgeContainer.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 	<div
-		class="tw-relative tw-inline-block tw-w-auto tw-overflow-hidden"
```

Comment:
`tw-overflow-hidden` is already used in TY page, can we add a prop for that case?

Reply:
Well, it's used for shining effect, probably should be added under that condition...

Reply:
Looks like fixed for all cases we have.

Is this case no longer available?
<img width="377" alt="Screenshot 2024-12-12 at 2 49 54 p m" src="https://github.com/user-attachments/assets/79cf0251-3f12-4eff-89ea-bf5576581989" />